network/bitcoin_transaction.ksy

```diff
@@ -76,8 +77,9 @@ types:
         size: script_len
         doc: |
           ScriptPubKey.
-        doc-ref: 'https://en.bitcoin.it/wiki/Transaction#Output'
-        doc-ref: 'https://en.bitcoin.it/wiki/Script'
+        doc-ref: |
+          'https://en.bitcoin.it/wiki/Transaction#Output'
```

Comment:
I guess you should remove the quotes here

Reply:
Done.